rs/cli/src/unit_tests/ctx_init.rs

```diff
@@ -4,17 +4,22 @@ use crate::{
     auth::{Auth, Neuron, STAGING_KEY_PATH_FROM_HOME, STAGING_NEURON_ID},
     commands::{AuthOpts, AuthRequirement, HsmOpts},
     cordoned_feature_fetcher::MockCordonedFeatureFetcher,
+    store::{Store, FALLBACK_IC_ADMIN_VERSION},
 };
 use clio::{ClioPath, InputPath};
 use ic_canisters::governance::governance_canister_version;
 use ic_management_backend::health::MockHealthStatusQuerier;
 use ic_management_types::Network;
 use itertools::Itertools;
 
-use crate::{commands::IcAdminVersion, ctx::DreContext, ic_admin::FALLBACK_IC_ADMIN_VERSION};
+use crate::{commands::IcAdminVersion, ctx::DreContext};
 
 fn status_file_path() -> PathBuf {
-    dirs::home_dir().unwrap().join("bin").join("ic-admin.revisions").join("ic-admin.status")
+    dirs::cache_dir()
+        .unwrap()
+        .join("dre-store")
```

Comment:
Can we have this directory in a single place only? E.g. there could be a class method that returns the local store path, and would be called both from the `Store` and from here.

Reply:
Tests are not finished, so this will change soon when I start adding tests for the agreed implementation